src/templates/mt5/economic_calendar.html.tt

```diff
@@ -0,0 +1,23 @@
+<div class="primary-color-dark economic-calendar-header">
+    <div class="fx-panel center-text container">
+        <h1 class="content-inverse-color">[% l('Economic Calendar') %]</h1>
+        <p class="content-inverse-color">[% l('Check out the latest economic events and indicators from all over the world. All data are collected and updated in real time.') %]</p>
+    </div>
+</div>
+
+<div class="container">
+    <p>[% l('You can monitor macroeconomic indicators straight from the MetaTrader 5 platform using release time labels displayed on <a href="https://www.metatrader5.com/en/terminal/help/fundamental?utm_source=www.mql5.com&utm_campaign=en.main.page.mql5#chart" target="_blank">financial symbol charts</a>.') %]</p>
```

Comment:
The link can be simply: `https://www.metatrader5.com/en/terminal/help/fundamental#chart`